tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -235,25 +235,35 @@ of the full release process.
 ### Prepare the release in the `flutter/devtools` repo
 
 1. Find the [DevTools tag](https://github.com/flutter/devtools/tags) that you want
-to perform the cherry-pick release on top of. Then checkout that tag locally. For this
+to perform the cherry-pick release on top of.
+
+For example, if you want current Flutter beta to be updated with fixed DevTools, run in Flutter repo:
```

Comment:
For example, if you want to perform a cherry pick release of DevTools into the Flutter / Dart beta branch, you can see what the current version of DevTools is on Flutter / Dart beta by doing the following:
- Go to https://flutter-ko.dev/development/tools/sdk/releases and copy the version of the latest Beta release.
- In your terminal from the `flutter` repository, run `git checkout <latest-beta-version>`.
- Once your local `flutter` source is on the latest beta, run `flutter --version` to see which version of DevTools is on Flutter / Dart beta. **This assumes that `which flutter` points to your local Flutter git repository.**